packages/desktop/src/renderer/components/CreateJoinCommunity/JoinCommunity/JoinCommunity.tsx

```diff
@@ -55,6 +61,9 @@ const JoinCommunity = () => {
       isConnectionReady={isConnected}
       isCloseDisabled={!currentCommunity}
       hasReceivedResponse={Boolean(currentIdentity && !currentIdentity.userCertificate)}
+      showPassword={showPassword}
+      handleClickShowPassword={handleClickShowPassword}
+      isPasswordField={true}
```

Comment:
Do we really need this prop? Can't we just distinguish basing on 'communityOwnership' ?